oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -167,6 +173,10 @@ public boolean close() {
             LOG.error("Error waiting for bulk requests to return", e);
         }
 
+        if (throwable != null) {
+            throw throwable;
```

Comment:
Instead of throwing Throwable, could you throw an IOException (or some other checked or runtime exception that makes sense in this context), and if "throwable" isn't such an exception (e.g. IOException), then wrap it: new IOException(throwable) ?